### ISSUE TITLE
Add TTY number to search results items

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#18480]
**[2025-01-24]** Blocked on #20343 

## Background
In an effort to move the facility locator redesign work forward, we are trying to break up the large project into smaller, more manageable pieces that can be completed more easily and added into sprints.

## Description 
Add TTY number to each search result item in the Facility Locator
   - The TTY number is necessary for accessibility, to allow folks with disability to have access to be able to call the facilities in the results listings.
   - [Link to Figma](https://www.figma.com/design/GvNVeG9GAXZaBpny7vNxZP/Facility-Locator?node-id=0-1&t=qGBDjLrP5HyJ7ZZK-0)

<details>
<summary>Screenshots</summary>

**Current display**
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/0b52b5e6-439c-4827-9433-0511f9846fac)

**New display**
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/6533cd4a-771e-49c1-91d7-cd7d3d593d08)

</details>

## Acceptance Criteria
- [x] For each results listing on the Facility locator, the TTY number is added as the last phone number - as shown in the Figma file
- [x] Requires design review
- [x] Requires accessibility review

**NOTE: this can ship to prod when it's done, does not need to be feature toggle gated / researched**

**Comment by Agile6MSkinner** [comment]
This is hardcoded since it's always 711

**Comment by jilladams** [comment]
From planning: we need to do this on Facility pages / phone numbers as well. That needs to be a separate ticket. (cc @Agile6MSkinner ) 

@laflannery @Shamiaalam we are thinking this should be an a11y audit to identify any other places where phone numbers need TTY.  We know of https://github.com/department-of-veterans-affairs/va.gov-cms/issues/20325, but beyond that: might need Vet Center, VBA, etc.

**Comment by Shamiaalam** [comment]
@jilladams I already discussed with @laflannery about this issues and for the priority based, please let me know if this issue counted as high priority, then I will try to find it as soon as possible but right now we don't need a11y audit. Only need to find pages where we need TTY number.

**Comment by jilladams** [comment]
@Shamiaalam  What I was trying to say: we will need to review all the Facility pages on VA.gov that come from Drupal, and identify any other standard phone number fields where TTY should be added. That's what I meant by audit. Is that what you meant re: "need to find pages where we need TTY number"?

**Comment by Shamiaalam** [comment]
yes, Exactly @jilladams thats what I meant to say.

**Comment by randimays** [comment]
@thejordanwood or @davidmpickett - I missed that this ticket required design review; my apologies. The code is in production; could one of you please verify the TTY number on all of the facility types and let me know if I've missed anything? Thanks.

**Comment by thejordanwood** [comment]
@randimays This looks great and I'm excited to see this change! I tested on desktop and mobile and don't believe you missed anything.

**Comment by jilladams** [comment]
Closing based on that feedback!

**Comment by randimays** [comment]
Thanks Jordan and Jill!
